packages/bot-web-ui/src/components/dashboard/tutorial-tab/sidebar.scss

```diff
@@ -0,0 +1,81 @@
+.dc-tabs {
+    &__wrapper {
+        padding: 1.6rem 0.8rem;
+
+        &__group {
+            display: flex;
+            width: 22.5rem;
+            position: relative;
+            justify-content: center;
+            align-items: center;
+            background: var(--general-section-6);
+            padding: 1.6rem 0;
+            z-index: 100;
+
+            svg {
+                position: absolute;
+                left: 1.5rem;
+                z-index: 1;
+            }
+
+            &--search_input {
+                width: 21rem;
+                height: 3.2rem;
+                border-radius: 0.4rem;
+                outline: none;
+                background-color: transparent;
+                font-size: 1.4rem;
+                padding-left: 3rem;
+                border: solid 1px #d6dadb;
```

Comment:
no const color for this? if not, we can add

Reply:
done ✅